dll/win32/windowscodecs/CMakeLists.txt

```
@@ -80,6 +85,6 @@ endif()
 
 set_module_type(windowscodecs win32dll)
 target_link_libraries(windowscodecs wine uuid ${PSEH_LIB})
```

```suggestion
target_link_libraries(windowscodecs wine uuid ${PSEH_LIB} oldnames)
```
(see my previous comment about "oldnames")